interface/modules/zend_modules/module/Carecoordination/src/Carecoordination/Model/EncounterccdadispatchTable.php

```diff
@@ -2192,62 +2197,88 @@ public function getCCDAComponents($type)
     * @param    integer     $status
     * @return   None
     */
-    public function logCCDA($pid, $encounter, $content, $time, $status, $user_id, $view = 0, $transfer = 0, $emr_transfer = 0)
+    public function logCCDA($pid, $encounter, $content, $time, $status, $user_id, $document_type, $view = 0, $transfer = 0, $emr_transfer = 0)
     {
         $content = base64_decode($content);
```

Comment:
if goal here is to use documents mechanism, then would not try to recreate it (it is complicated, considering couchdb especially, encryption, and uuid, and also then need to updated it when the core document mechanism is updated). Instead should just run it through the document function (createDocument()), which does it all via the centralized function.

Reply:
oh wait, you are using createDocument. I was thrown off by the creation of the uuid which looks like you are doing for the ccda table entry